projects/ngx-formentry/src/components/ngx-remote-select/ngx-remote-select.component.ts

```diff
@@ -138,17 +138,22 @@ export class RemoteSelectComponent implements OnInit, ControlValueAccessor {
 
   private loadOptions() {
     this.remoteOptions$ = concat(
-      of([]), // default items
+      this.dataSource.searchOptions(
+        '',
+        this.dataSource?.dataSourceOptions ?? {}
+      ) ?? of([]), // default items
```

Comment:
```suggestion
      this.dataSource.searchOptions(
        '',
        this.dataSource?.dataSourceOptions ?? {}
      )?.pipe(
        catchError((error) => {
          console.error('Error loading initial options:', error);
          return of([]);
        })
      ) ?? of([]),
```

Might be useful to add some kind of error handling.